zkvm-host/bin/agg.rs

```diff
@@ -0,0 +1,123 @@
+use std::fs;
+
+use anyhow::Result;
+use cargo_metadata::MetadataCommand;
+use clap::Parser;
+use client_utils::{types::AggregationInputs, RawBootInfo, BOOT_INFO_SIZE};
+use host_utils::fetcher::{ChainMode, SP1KonaDataFetcher};
+use sp1_sdk::{utils, ProverClient, SP1Proof, SP1ProofWithPublicValues, SP1Stdin};
+
+pub const AGG_ELF: &[u8] = include_bytes!("../../elf/aggregation-client-elf");
+pub const MULTI_BLOCK_ELF: &[u8] = include_bytes!("../../elf/validity-client-elf");
+
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+struct Args {
+    /// Start L2 block number.
+    #[arg(short, long, num_args = 1.., value_delimiter = ',')]
+    proofs: Vec<String>,
+
+    #[arg(short, long)]
+    multi: bool,
+
```

Comment:
This argument is never used. I think safe to assume we'll just use agg for multi.